code/.yarnrc.yml

```
@@ -37,4 +37,4 @@ supportedArchitectures:
 unsafeHttpWhitelist:
   - localhost
 
-yarnPath: .yarn/releases/yarn-4.6.0.cjs
+yarnPath: .yarn/releases/yarn-4.8.1.cjs
```

We could re-use the installation 1 folder up:
```suggestion
yarnPath: ../.yarn/releases/yarn-4.8.1.cjs
```
And then remove the installation in the 2 sub directories?

---

Yeah,,, there are a few things I came across like this that IMO could be streamlined - but trying to keep the changeset small for now? 

e.g. we could remove `yarnPath` in favor of fully adopting corepack in CI (right now it's only partially enable)